### PR TITLE
Add tutorial how to install PostgreSQL and pgAdmin with docker and connect pgadmin with postgres

### PR DESCRIPTION
**Pull request by PatrickWe**
I have read and understood the Contributor's Certificate of Origin at the end of the template and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: (Patrick Weber, netcup_wiki@icloud.com)

### CODE REVIEW

File: community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+networks:
+  backend_network:
+```
+
+Save the compose.yaml file.
+
+Here are two services defined: One for PostgreSQL and one for pgAdmin. Both services have a fix container-name and both have its own volume. In the environments-definition you can see the credentials to access the services. The postgres servuce will be exposed on port 5432 to the outside. pgadmin will be exposed on port 8080 to the outside. Both services are using the same docker network "backend_network", which is also defined in the networks-definition.
+
+# Step 5 - Start all docker services
+To start all docker services run: ```docker-compose up -d```
```
Comment (d-schaffer) [inline community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md:74]: ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?
        Supported filenames: **docker-compose.yml, docker-compose.yaml**

Comment (d-schaffer) [reply community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md:74]: if i rename the file like "docker-compose.yaml"
i get another error:
```
Unsupported config option for networks: 'backend_network'
Unsupported config option for services: 'db'
Unsupported config option for volumes: 'db_volume'
```

**Review by d-schaffer — CHANGES_REQUESTED** [review]
need help, can't even start the container according to the description
Thx :)

File: community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md (hunk 1 above)
Comment (PatrickWe) [reply community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md]: i moved the file from compose.yaml to docker-compose.yaml.
I also added the version: '3' in the docker-compose-file, to support older versions.

Can you pull the latest changes and try the docker-compose file again?

File: community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+The log should look like:
+```
+[2021-11-07 21:21:35 +0000] [1] [INFO] Starting gunicorn 20.1.0
+[2021-11-07 21:21:35 +0000] [1] [INFO] Listening at: http://[::]:8080 (1)
+[2021-11-07 21:21:35 +0000] [1] [INFO] Using worker: gthread
+[2021-11-07 21:21:35 +0000] [88] [INFO] Booting worker with pid: 88
+```
+
+# Step 8 - Open pgadmin in your browser
+Open the url ```http://localhost:8080/``` in the browser.
```
Comment (d-schaffer) [inline community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md:133]: Please change localhost to "your-ip" or something like that

File: community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+Check the docker-compose version:
+```
+docker-compose version
+```
+Output should look like this, maybe with another version:
+```
+Docker Compose version v2.1.0
+```
+
+# Step 4 - Create a docker-compose.yaml file, with PostgreSQL and pgAdmin
```
Comment (d-schaffer) [inline community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md:41]: Please check the numbering

**Review by d-schaffer — CHANGES_REQUESTED** [review]
very good thank you. there are only 2 small changes left

File: community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md (hunk 3 above)
Comment (PatrickWe) [reply community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md]: done, i pushed the changes

File: community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md (hunk 2 above)
Comment (PatrickWe) [reply community-tutorials/how-to-install-postgres-and-pgadmin-with-docker-and-connect-pgmadmin-with-postgres/01-en.md]: done, i pushed the changes

**Comment by netcupClaudiaM** [comment]
Hi PatrickWe, thank you for contributing! It's been a while since you opened this PR and it probably is an open secret we're currently overwhelmed with reviewing all the awesome tutorials we got. Please be patient - we're about to publish tutorials this and next week. Thank you!

**Comment by PatrickWe** [comment]
Please delete all of my opened pull requests. It is outrageous, that vouchers are not allowed for more than the half of your products. Shame on you.

**Pull request closed without merging**